app/controllers/app.controllers.js

```diff
@@ -16,4 +20,13 @@ const getArticles = (request, response, next) => {
     .catch(next);
 };
 
-module.exports = { getTopics, getArticles };
+const getArticleById = (request, response, next) => {
+  const articleId = request.params;
```

Comment:
Might be better to destructure here - as its names article_id, but is actually an object of params, one of which is article id